### PR TITLE
[290] Add more multioutput tests and abstract the scenario

### PR DESCRIPTION
**Pull request by paweljakubas**
<p align="right">#190</p>

# Overview

- [x] I have added more tests to multioutput transaction section and abstracted them 

# Comments

The results are along with expectations in all boundary conditions

### CODE REVIEW

**Comment by KtorZ** [comment]
@paweljakubas 

Are you referring to this one (your incomprehension):

```
        [60000, 60000, 60000]
        [10,11]
        [ expectWalletError (NotEnoughMoney ErrCannotCoverFee) ]
```

 ?

**Comment by paweljakubas** [comment]
This also :  
```
[90000, 90000]
[10,11]
[ expectWalletError (NotEnoughMoney ErrCannotCoverFee) ]
```

**Comment by paweljakubas** [comment]
I underestimated the fees in those transactions. Now everything good

File: test/integration/Test/Integration/Scenario/Transactions.hs [hunk 1]
```diff
@@ -349,3 +340,39 @@ spec = do
             verify response
                 [ expectTxStatusEventually [InNewestBlocks, Persisted]
                 ]
+
+    multioutputTransactionScenario
+        :: String
+        -> [Word64]
+        -> [Word64]
```
Comment (KtorZ) [inline test/integration/Test/Integration/Scenario/Transactions.hs:347]: Again, this is a bit misleading since the underlying implementation only uses (and actually expects exactly) two elements in the outputDistribution list. 
Moving on merging _anyway_ since this is a local function. But if you feel like making another PR to fix that and make it a bit more type-safe, I'll fall sleep very happy tonight :wink:

**Review by KtorZ — APPROVED** [review]
Good testing scenarios here, this is useful :pray:
